core/src/main/java/apoc/cypher/CypherInitializer.java

```diff
@@ -142,10 +142,20 @@ public void available() {
                 final boolean isSystemDatabase = db.databaseName().equals(GraphDatabaseSettings.SYSTEM_DATABASE_NAME);
                 Configuration config = dependencyResolver.resolveDependency(ApocConfig.class).getConfig();
                 var initializers = collectInitializers(isSystemDatabase, config);
+
+                // Wait for database to become available, once it is available
+                // procedures are also available by necessity.
```

Comment:
In 5.x, this comment was for `while (!db.isAvailable(100));` so I wonder if we really should add it here?